.dockerignore

```diff
@@ -7,6 +7,8 @@ golang/cosmos/build
 packages/deployment
 packages/swingset-runner
 packages/stat-logger
+packages/swingset-xsnap-supervisor/dist
+packages/xsnap-lockdown/dist
```

Comment:
we have `**/build` and `**/bundles`. Is there a reason not to `**/dist`?

Reply:
There is one: `packages/cosmic-proto/dist` ...

Reply:
would be helpful to comment here with that reason. not a blocker.